tox.ini

```diff
@@ -1,19 +1,28 @@
 [tox]
 envlist =
     py{27,35}-codestyle,
-    py{27,35}-django{18,19,110}
+    py{27,35}-django{18,19,110,111}
 
 [testenv]
 setenv =
     PYTHONDONTWRITEBYTECODE=1
-install_command = pip install --no-deps {opts} {packages}
+install_command = pip install {opts} {packages}
```

Comment:
I do keep the requirements used in tests up to date and make updates if any changes are required... https://github.com/ottoyiu/django-cors-headers/commits/master